src/app/webcontainer/WebappSettingsPage.qml

```diff
@@ -49,6 +49,26 @@ FocusScope {
 
                 width: parent.width
 
+                ListItem {
+                    objectName: "autoFitToWidthEnabled"
+
+                    ListItemLayout {
+                        title.text: i18n.tr("Automatic Fit to Width")
```

Comment:
Other items on this page are sentence cased, not Title Cased. Please follow that convention.

Reply:
Sorry, got distraced by the ```Set Desktop mode``` casing. Changed to sentence cased in both settings pages (browser, container)